src/event.ts

```diff
@@ -130,7 +130,7 @@ interface EventDamage extends FieldsTargeted {
 	amount: number
 	/** Amount of total damage that was overkill. */
 	overkill: number
-	/** Unique numeric ID that will match this damage to the snapshot it stems from. If omitted, no snapshot was performed (status ticks, etc). */
+	/** Unique numeric ID that will match this damage to its execution confirmation. If omitted, no confirmation occurred (status ticks, etc). */
```

Comment:
nit; "no confirmation **will occur**", given it's looking forward rather than back